docs/release/release-process.md

```diff
@@ -185,7 +185,7 @@ examples to point to the new versions that we will be releasing soon.
   guides.
   * searching for `:master` will help a lot here
 * Examples, such as [`StackInstall` yaml
-  files](https://github.com/crossplane/crossplane/tree/release-0.9/cluster/examples/provider)
+  files](https://github.com/crossplane/crossplane/tree/release-0.9/examples/provider)
```

Comment:
Could you omit this update as the existing path is still correct for the release-0.9 branch? 🙂

Reply:
@hasheddan Will do. Do you want me to amend the commit and force push it OR make another commit?
Not sure what is the git workflow your org follows. Hence the Q.

Reply:
Force push is good 👍

Reply:
@hasheddan The changes have been force pushed to https://github.com/nchetan/crossplane. Do I have to re-request?